torch/cuda/amp/grad_scaler.py

```diff
@@ -396,6 +396,12 @@ def update(self, new_scale=None):
         .. warning::
             :meth:`update` should only be called at the end of the iteration, after ``scaler.step(optimizer)`` has
             been invoked for all optimizers used this iteration.
+
+        .. warning::
+            For performance reasons, the scale factor is not guaranteed to be above 1. If the
```

Comment:
add here that for performance reasons we are not checking scale factor value to avoid synchronization, so it is not guaranteed to be above 1.